src/Symfony/Component/Translation/Bridge/PoEditor/PoEditorProviderFactory.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Bridge\PoEditor;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Translation\Exception\UnsupportedSchemeException;
+use Symfony\Component\Translation\Loader\LoaderInterface;
+use Symfony\Component\Translation\Provider\AbstractProviderFactory;
+use Symfony\Component\Translation\Provider\Dsn;
+use Symfony\Component\Translation\Provider\ProviderInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class PoEditorProviderFactory extends AbstractProviderFactory
+{
+    public const SCHEME = 'poeditor';
```

Comment:
in mailer and notifier we don't use a const for the scheme

Reply:
You fix that in https://github.com/symfony/symfony/pull/40945? I will rebase the PR to fix it on PoEditor Provider and Lokalise Provider. @andrii-bodnar can you also rebase your PR when #40945 will be merged and update Crowdin Provider?

Reply:
Created a separate PR https://github.com/symfony/symfony/pull/40967

just FYI